docs/references/nextjs/route-protection-requiring-authentication.mdx

```diff
@@ -0,0 +1,236 @@
+---
+title: Requiring authentication for route protection | Next.js
+description: Learn how to protect the routes in your Next.js application that require authentication.
+---
+
+# Requiring authenication for route protection
+
+Whether you're using the App Router, Pages Router, or Middleware, Clerk provides access to a standardized [`auth()`](/docs/references/nextjs/authentication-object) object that you can use to access authentication details throughout your application. You can find details below on how to access and use the `auth()` object.
```

Comment:
This description would make sense on the `Auth` object page, but this entire guide is about requiring authentication for route protection - it's not necessarily about "how to access and use the Auth object". I think this summary could use some work - would you like help writing it?

Reply:
```suggestion
Whether you're using the App Router, Pages Router, or Middleware, Clerk provides access to a standardized [`auth()`](/docs/references/nextjs/authentication-object) object that you can use to access authentication details throughout your application. You can find details below on how to protect routes using the `auth()` object.
```

Reply:
```suggestion
Whether you're using the App Router, Pages Router, or Middleware, there are several methods you can use to protect routes within your app that require a logged in user to access. This guide will go through a variety of examples of how to do this.
```